cli/core/pkg/artifact/local_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package artifact
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLocalArtifact(t *testing.T) {
+	assert := assert.New(t)
+
+	// When local artifact directory exists and test directory exists
```

Comment:
nit-ish: IMO we should make these helper functions instead of commenting out what we expect from the subsequent lines